examples/pytorch/multigpu/README.md

```diff
@@ -0,0 +1,38 @@
+Multiple GPU Training
+============
+
+Requirements
+------------
+
+```bash
+pip install torchmetrics
+```
+
+How to run
+-------
+
+### Graph property predication
+
+
+Run with following (available dataset: "ogbg-molhiv", "ogbg-molpcba")
+```bash
+python3 multi_gpu_graph_prediction.py --dataset ogbg-molhiv
+```
+
+Results:
+```
+* ogbg-molhiv: ~0.7965
+* ogbg-molpcba: ~0.2239
+```
```

Comment:
Can you add your scalability result as a table in this README?

Reply:
sure, add it for already